CODE_OF_CONDUCT.rst

```diff
@@ -42,7 +42,7 @@ As members of the community,
 * We pledge to help the entire community follow the code of conduct, and to
   not remain silent when we see violations of the code of conduct. We will
   take action when members of our community violate this code such as
-  contacting einsteinpy.project@gmail.com (all emails sent to this
+  contacting shreyas@einsteinpy.org (all emails sent to this
   address will be treated with the strictest confidence) or talking privately
   with the person.
```

Comment:
We pledge to help the entire community follow the code of conduct, and to not remain silent when we see violations of the code of conduct. We will take action when members of our community violate this code such as contacting shreyas@einseinpy.org (all emails sent to this address will be treated with the strictest confidence) or talking privately with the person.

Reply:
Kindly replace these lines with the ones I suggested.